packages/contracts/contracts/ChugSplashManager.sol

```diff
@@ -349,13 +379,45 @@ contract ChugSplashManager is Owned {
         // completed and reset the active bundle hash so that a new bundle can be executed.
         if (bundle.total == bundle.executions.length) {
             bundle.status = ChugSplashBundleStatus.COMPLETED;
+            bytes32 completedBundleId = activeBundleId;
             activeBundleId = bytes32(0);
 
-            emit ChugSplashBundleCompleted(activeBundleId, msg.sender, bundle.total);
+            emit ChugSplashBundleCompleted(completedBundleId, msg.sender, bundle.total);
```

Comment:
Changed the event argument from `activeBundleId` to `completedBundleId` because `activeBundleId` is always bytes32(0) here.